sql/string-functions.md

```diff
@@ -0,0 +1,74 @@
+---
+title: 字符串函数
+category: user guide
+---
+
+# 字符串函数
+
+| 函数名                                                                                                              | 功能描述                                                                                                                               |
+|:------------------------------------------------------------------------------------------------------------------|:------------------------------------------------------------------------------------------------------------------------------------------|
+| [`ASCII()`](https://dev.mysql.com/doc/refman/5.7/en/string-functions.html#function_ascii)                         | 返回最左字符的数值                                                                                               |
+| [`CHAR()`](https://dev.mysql.com/doc/refman/5.7/en/string-functions.html#function_char)                           | 返回一个包含整数的代码值所给出的字符的字符串                                                                                              |
+| [`BIN()`](https://dev.mysql.com/doc/refman/5.7/en/string-functions.html#function_bin)                             | 返回一个数的二进制值的字符串表示                                                                              |
+| [`HEX()`](https://dev.mysql.com/doc/refman/5.7/en/string-functions.html#function_hex)                             | 返回十进制值或字符串值的十六进制表示                                                                          |
+| [`OCT()`](https://dev.mysql.com/doc/refman/5.7/en/string-functions.html#function_oct)                             | 返回一个数的八进制值的字符串表示                                                                               |
+| [`UNHEX()`](https://dev.mysql.com/doc/refman/5.7/en/string-functions.html#function_unhex)                         | 返回 HEX 表示的数字所代表的字符串                                                                                 |
+| [`TO_BASE64()`](https://dev.mysql.com/doc/refman/5.7/en/string-functions.html#function_to-base64)                 | 返回转换为基数为 64 的字符串参数                                                                                         |
```

Comment:
BASE64 is a technical term for the string translating algorithm. no need to split and translate